scaper/core.py

```diff
@@ -198,7 +238,22 @@ def generate_from_jams(jams_infile, audio_outfile, fg_path=None, bg_path=None,
 
     # Optionally save new jams file
     if jams_outfile is not None:
-        jam.save(jams_outfile)
+        soundscape_jam.save(jams_outfile)
+
+    # Create annotation list
+    annotation_list = []
+    for obs in ann.data:
+        if obs.value['role'] == 'foreground':
+            annotation_list.append(
+                [obs.time, obs.time + obs.duration, obs.value['label']])
+
+    if txt_path is not None:
+        with open(txt_path, 'w') as csv_file:
+            writer = csv.writer(csv_file, delimiter=txt_sep)
+            writer.writerows(annotation_list)
+
+    # Return
```

Comment:
Do we need this comment?

Reply:
Haha I like breaking down the structure of a function with comments, including the return, but no, we don't really need this comment.